src/lib/deploy.js

```diff
@@ -347,7 +350,8 @@ async function deploy({ alias, yes }) {
         `  Failed to find the fee payer's account on chain.\n  Please make sure the account "${zkAppAddressBase58}" has previously been funded.`
       )
     );
-    await shutdown();
+
+    process.exit(1);
```

Comment:
interesting that you needed these 😮 do you know if this is snarkyjs-related or something to do with yargs etc?